packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -348,11 +354,9 @@ export class PostPopovercontainer {
   }
 
   render() {
-    const animationClass = this.animation ? `animate-${this.animation}` : '';
```

Comment:
Before the animation only ran when `this.animation` was not null. Can you add this check in you changes as well?

Reply:
This issue seems not to be resolved. The `popIn` animation on line 211 is started even when `this.animation` is `undefined`.